src/Entity/Deputy.php

```diff
@@ -58,13 +60,15 @@ class Deputy
      * @var \DateTime
      *
      * @ORM\Column(name="dob", type="date", nullable=true)
+     * @Assert\NotBlank(message="deputy.dateOfBirth.notBlank")
```

Comment:
For all of these where we refer to `message="<SOMETHING>"` you'll need to add the corresponding translation to `/translations/validators.en.yml`.